infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -164,16 +182,16 @@ private void setDataConsistentWithRetry(boolean isDataConsistent) {
                 try {
                      metadataManager.setDataConsistentOnSink(isDataConsistent, session);
                 } catch (TransactionAbortedException tae) {
-                    log.error("Error while attempting to setDataConsistent in SinkManager's init", tae);
+                    log.error("[{}]:: Error while attempting to setDataConsistent in SinkManager's init", sessionName, tae);
```

Comment:
to print the exception in case of an error or warning, the logger takes a single argument only.  Now as we are adding the session name, it cannot be a separate argument.

From logger api class:
void warn(String var1, Throwable var2);
void error(String var1, Throwable var2);

Apologies if my previous comment caused the confusion.  It will not be applicable in cases where the exception stack trace is to be printed.

Reply:
There are APIs like:
    void error(String var1, Object var2);
    void error(String var1, Object var2, Object var3);
    void error(String var1, Object... var2);

These 'objects' are taken as args and printed.